apps/web/src/views/Swap/hooks/useFeeSaved.ts

```diff
@@ -8,20 +8,9 @@ import { useMemo } from 'react'
 const MAJOR_DEFINED_STABLE_COINS = {
```

Comment:
Let's add explicit type definition here. So we can identify the error at build time

Reply:
also can remove the optional mark in line 24

Reply:
explicit mark type as `ERC20Token | undefined` and keep optional mark, as `Token[chainId]` will not throw any warning during static check or build time.

Reply:
hmm ok that's weird. Let's keep it for now. Will double check later on